sites/next.skeleton.dev/src/content/docs/get-started/figma.mdx

```diff
@@ -1,26 +1,57 @@
 ---
 layout: '@layouts/LayoutDoc.astro'
 title: Figma
-description: A Figma UI Kit for Skeleton.
+description: The Figma UI Kit for Skeleton.
 order: 60
 ---
 
 export const components = componentSet;
 
-The Skeleton Figma UI Kit has been designed in coordination between [Bohdan Karplyak](https://www.linkedin.com/in/bohdan-karplyak-659a41162/), [Micha Mailänder](https://www.linkedin.com/in/micha-mail%C3%A4nder-548aba16a/), and [Skeleton Labs](https://www.skeletonlabs.co/).
+import FigmaBuyNow from '@components/docs/FigmaBuyNow.astro';
+import NavGrid from '@components/docs/NavGrid.astro';
 
-<img
-	src="https://github.com/skeletonlabs/skeleton/assets/1509726/956e6893-062e-4030-ba6a-d7b6e8acc4e8"
-	alt="figma-preview"
-	class="rounded-container"
-/>
+import { Image } from 'astro:assets';
+import ImgHero from '@images/get-started/figma/hero.webp';
+import ImgComponents from '@images/get-started/figma/components.webp';
+import ImgPlugin from '@images/get-started/figma/plugin.webp';
+import ImgTutorial from '@images/get-started/figma/tutorials.webp';
+
+<div class="relative">
+	<Image src={ImgHero} class="rounded-container" alt="hero" />
+	<a
+		class="btn btn-lg preset-filled-primary-500 flex items-center gap-2 whitespace-nowrap absolute top-6 right-6 shadow-lg w-48"
+		href="#get-access"
+	>
+		Get Access
+	</a>
+</div>
+
+Created and managed by [Bohdan Karpliak](https://www.linkedin.com/in/bohdan-karpliak-659a41162/) and [Micha Mailänder](https://www.linkedin.com/in/micha-mail%C3%A4nder-548aba16a/) in collaboration with [Skeleton Labs](https://www.skeletonlabs.co/).
+
+## Start designing directly
+
+Save weeks of preparation work. We have crafted every Skeleton component plus other helpful components in detail for you, so you can focus completely on customer value.
+
+<Image src={ImgComponents} class="rounded-container" alt="components" />
+
+## Make it your own
+
+Import your custom Skeleton Theme with our Figma plugin and let it shine in your design.
+
+<Image src={ImgPlugin} class="rounded-container" alt="plugins" />
+
+## Pair Figma with Skeleton.
+
+From setting-up Skeleton in Figma to creating wireframes and pixel perfect design, we cover your back.
+
+<Image src={ImgTutorial} class="rounded-container" alt="tutorials" />
 
 ## Get Access
 
-> Maintainer Note: this will be made available alongside the stable release of Skeleton v3.
+Support the development of Skeleton by purchasing the Figma UI Kit! Your purchase helps ensure continued development and success of Skeleton as free and open-source project.
```

Comment:
I'm a little leery of how this information is presented. Obviously the Figma designs now longer directly benefit Skeleton. I might suggest leaving that more understate, if not removing the line altogether. Less is more some times.